lib/tests/formBuilder.test.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { cleanup, render, screen } from "@testing-library/react";
 import { Form } from "../../components/forms";
 import { getProperty, getFormInitialValues, getRenderedForm } from "../formBuilder";
-import TestForm from "./testData";
+import { testForm } from "./testData";
```

Comment:
Is their any particular reason for this alteration ?

Reply:
"TestForm" was exported as default from the module. I added another variable then i change them in lowercase to make  the export consistent with the variable name.

Reply:
You could still have a default export while exporting other variables.